Ebraheem-Mahmood/index.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html>
+    <head>
+        <title>twitter</title>
+        <meta name="description" content="Our first page">
+        <meta name="keywords" content="html tutorial template">
+        <link href="style.css" rel="stylesheet" type="text/css">
+    </head>
+    <body>
+        <header>
+
+            <div id="logo-side">
+                <img src="https://seeklogo.net/wp-content/uploads/2015/11/twitter-logo.png" alt="twiiter" class="logo-img">
+            </div>
+            <div class="btn-side">
+                <a href="twitter/login" class="btn-white">login</a>
+                <a href="twitter/register" class="btn-blue">register new account</a>
+            </div>
+        </header>
+        <main>
```

Comment:
Great job on using Semantic HTML! It makes reading and maintaining your code a lot easier. 🙂